src/constructs/iam/policies/log-shipping.test.ts

```diff
@@ -0,0 +1,94 @@
+import "@aws-cdk/assert/jest";
+import { Role, ServicePrincipal } from "@aws-cdk/aws-iam";
+import { App } from "@aws-cdk/core";
+import { GuStack } from "../../core";
+import { GuLogShippingPolicy } from "./log-shipping";
+
+describe("The GuLogShippingPolicy class", () => {
+  it("sets default props", () => {
+    const stack = new GuStack(new App());
+
+    const logShippingPolicy = new GuLogShippingPolicy(stack, "SSMRunCommandPolicy", { kinesisStreamName: "test" });
```

Comment:
Tiny one but there's a copy and paste error in the id here

```suggestion
    const logShippingPolicy = new GuLogShippingPolicy(stack, "LogShippingPolicy", { kinesisStreamName: "test" });
```